src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/event/EventToSendProcessor.java

```diff
@@ -58,4 +59,38 @@ public int processChunk() {
     }
     return numProcessed.get();
   }
+
+  /**
+   * We need another class for the retryable annotation, since calling a retryable annotated within
+   * the same class does not honour the annotations.
+   */
+  @Slf4j
+  @Component
+  static class PublishRetrier {
+    private EventPublisher eventPublisher;
+    private PublisherRetryConfig retryConfig;
+
+    public PublishRetrier(EventPublisher eventPublisher, AppConfig appConfig) {
+      this.eventPublisher = eventPublisher;
+      this.retryConfig = appConfig.getMessaging().getRetry();
+      log.info("Publish-retry configuration: {}", this.retryConfig);
+    }
+
+    @Retryable(
+        label = "publishEvent",
+        include = EventPublishException.class,
+        backoff =
+            @Backoff(
+                delayExpression = "#{@appConfig.getMessaging().getRetry().getInitial()}",
+                multiplierExpression = "#{@appConfig.getMessaging().getRetry().getMultiplier()}",
+                maxDelayExpression = "#{@appConfig.getMessaging().getRetry().getMax()}"),
+        maxAttemptsExpression = "#{@appConfig.getMessaging().getRetry().getMaxAttempts()}",
+        listeners = "publishRetryListener")
+    public UUID publish(EventToSend event) {
```

Comment:
I think the `@Scheduled` uses a `ScheduledExecutorService` that runs on a singled thread, so no worry about the health indicators since they will have the normal HTTP pool of threads to choose from